Release_Guideline.md

```diff
@@ -0,0 +1,38 @@
+# Release Guidelines
+
+This document captures guidelines to follow when considering a new release of the BIDS specification.
+
+## Background
+BIDS has generally followed a semantic versioning (semver) approach.
+The canonical semver text can be found at [https://semver.org/](https://semver.org/), and an adaptation to documents can be found at [https://semverdoc.org/](https://semver.org/).
+A specification falls somewhere between these two, as documentation does not require
+backwards compatibility, while software has notions of bugs and features which do not
+map to the reasons for updating a specification.
+
+In both cases, edition numbers are specified as MAJOR.MINOR.PATCH, and the difference is in the rules for incrementing each edition number.
+
+In BIDS, we have considered a major release version (the next being 2.0.0) to indicate
+backwards-incompatible changes, while minor and patch releases must be backwards compatible.
+## Guidelines
+
+Once a decision for a release has been established, the rules of [decision-making](DECISION-MAKING.md)
+govern the mechanism of doing the release, *i.e.*, waiting 5 business days to merge.
```

Comment:
```suggestion
govern the mechanism of doing the release, *i.e.*, waiting 5 business days and obtain approval by at least one maintainer to merge.
```

as taken from https://github.com/bids-standard/bids-specification/blob/master/DECISION-MAKING.md

Reply:
Verb agreement and wrapping the line.

```suggestion
govern the mechanism of doing the release, *i.e.*, waiting 5 business days and obtaining
the approval of at least one maintainer before merging.
```